applications/ParticleMechanicsApplication/custom_utilities/mpm_search_element_utility.h

```diff
@@ -85,6 +88,42 @@ namespace MPMSearchElementUtility
                 // FindPointOnMesh find the background element in which a given point falls and the relative shape functions
                 bool is_found = SearchStructure.FindPointOnMesh(xg[0], N, pelem, result_begin, MaxNumberOfResults, Tolerance);
 
+
+                if (is_found && isExplicit) {
+                    // check if MP is exactly on the edge of the element, this gives spurious strains in explicit
+                    bool isOnEdge = false;
+                    for (SizeType i = 0; i < N.size(); ++i) {
+                        if (abs(N[i]) < std::numeric_limits<double>::epsilon()) {
```

Comment:
this is a bug in linux! (Google will tell you why ;) )
use std::abs

Reply:
Fixed